src/Razor/src/Microsoft.AspNetCore.Razor.Language/TagHelperMatchingConventions.cs

```diff
@@ -138,6 +140,18 @@ public static bool SatisfiesBoundAttributeIndexer(string name, BoundAttributeDes
                 name.StartsWith(descriptor.IndexerNamePrefix, StringComparison.OrdinalIgnoreCase);
         }
 
+        public static bool SatisfiesBoundAttributeParameter(string name, BoundAttributeDescriptor parent, BoundAttributeParameterDescriptor descriptor)
```

Comment:
`SatisfiesBoundAttributeWithParameter` as it is currently it sounds like it's just validating the parameter portion.